conf/app.routes

```diff
@@ -5,5 +5,7 @@ POST        /scratch              controllers.ScratchController.save
 
 GET         /published/:id        controllers.PublishedController.get(id)
 
+GET         /approval/list        controllers.ApprovalController.approvalSummaryList
```

Comment:
prefer /approval without list, so /approval/:id for specific, /approval for list

Reply:
updated to /approval